### ISSUE TITLE
Workspace not registered after upgrade to 6.5.0

### ISSUE THREAD

**Issue by kaidohTips** [RocketChat/Rocket.Chat#31157]
Hello everyone,

I am using a Docker environment, and everything worked fine with version 6.4.6. However, yesterday, I upgraded to version 6.5.0, and now, all my enterprise features are missing. I can no longer resync the workspace because the dashboard has changed.

Am I missing something ?

![screenshoot_2023-12-05_1](https://github.com/RocketChat/Rocket.Chat/assets/77806690/9878f155-a935-4f9e-bc28-df62bbd9ded8)

**Comment by caskelly** [comment]
@imYurikhan  @marceloschmidt - I am tagging Marcelo Schmidt here to assist because your post indicates that you are an enterprise customer.

**Comment by marceloschmidt** [comment]
@imYurikhan could you please open a ticket in https://desk.rocket.chat if you haven't done so? If you have, please share your ticket number.

**Comment by imYurikhan** [comment]
@caskelly  @marceloschmidt  Thank you for your prompt response; here is the ticket number [#82849]

**Comment by xet7** [comment]
Hi,
I'm maintainer of WeKan, and use RocketChat snap package https://snapcraft.io/rocketchat-server that just updated to version 6.5.0, Rev 2597, 6.x/stable .

Now when trying to go to login, there is page /setup-wizard/3 where registering with email address does not work. I do receive that registering email, copy link to registering page tab, and it requires to register again and again.

<img width="957" alt="rocketchat" src="https://github.com/RocketChat/Rocket.Chat/assets/15545/f97e7a58-7e36-4ae6-8644-cdb73bc90db0">

**Comment by xet7** [comment]
So currently it is not possible to login to RocketChat at all.

**Comment by caskelly** [comment]
@geekgonecrazy tagging you here.  We are not seeing this issue elsewhere as far as I know.  Can you help to troubleshoot this issue.

**Comment by xet7** [comment]
Registering did not work at WaterFox, that is fork of Firefox. I now tried at Chrome, registering did work, and I did get email "A 30-day Pro trial has begun on your Rocket.Chat workspace". I had to insert my RocketChat account password many times when login to RocketChat, and then it allowed me to login to RocketChat, both at Chrome and WaterFox. But I don't know why upgrading did require signing up for trial. And I don't know what happens after trial or how to cancel trial.

**Comment by geekgonecrazy** [comment]
Does it take you past the registration screen?

The flow should be:
1. You put in info on that screen
2. You get email to confirm registration and you see a code that matches what’s in Rocket.chat
3. You click the link let it open in new tab.  It’ll say confirmed
4. The existing tab you’ll then see it continue as well and complete registration 


I suspect based on description might be pasting into Rocket.chats window the link from email.  But that tab needs to stay open.  What’s happening is it’s polling waiting for a confirmation.  Once you click the link in email it gets back a success and then finishes the handshake.  

But if you replaced the tab nothing is polling and so handshake never gets finished.

The flow is similar in ways to how you do a device register on your tv to login to Netflix or Hulu or any of those tv apps where you confirm via another device.

**Comment by xet7** [comment]
@geekgonecrazy 

Yes, after all those steps using Chrome, I did get past registration screen and was able to login again.

**Comment by geekgonecrazy** [comment]
Great! Bringing this up with the team as we should potentially make more clear that the page needs to stay open and then in email maybe say it should open in new tab.

I think I saw your user on open.  I'll shoot you a DM and we can help clean up the duplicate stuff so you don't get spammed later

**Comment by TheDom42** [comment]
> Registering did not work at WaterFox, that is fork of Firefox. I now tried at Chrome, registering did work, and I did get email "A 30-day Pro trial has begun on your Rocket.Chat workspace". I had to insert my RocketChat account password many times when login to RocketChat, and then it allowed me to login to RocketChat, both at Chrome and WaterFox. But I don't know why upgrading did require signing up for trial. And I don't know what happens after trial or how to cancel trial.

@xet7 Check out https://github.com/RocketChat/Rocket.Chat/issues/31149 - there is a discussion on the enterprise features and how to "downgrade" back to the Community Edition.

**Comment by Gummikavalier** [comment]
We met a same or similar issue with our Enterprise instance we try to change into an offline licensed one.

The previous cloud based registration has been set to inactive to be replaced by the new offline one. The wizard triggered on our instance but it did not include an offline button option. So I had to register it to Enterprise trial for now.

Now I'm struggling to find any official method to trigger the offline-registration wizard to complete our license installation.

The same issue happens on the second RC instance where we are moving from CE to Enterprise. This one cannot connect to cloud.rocket.chat.

So in short there are two problems:
1. The registration page /setup-wizard/3 does not show the `Register offline` button depicted here: https://docs.rocket.chat/setup-and-configure/rocket.chat-air-gapped-deployment/offline-workspace-registration
2. The admin settings themselves do not contain any button to enforce `Register offline` method.

**Comment by imYurikhan** [comment]
Hello @Gummikavalier, @marceloschmidt resolve my issue soon as I address a ticket to the rocketchat support. I suggest you to do the same.

**Comment by Gummikavalier** [comment]
Regarding my problem above, it was just that Offline registration setting had changed its name to Premium and can now be found under `Admin -> Settings -> Premium`. Doh! 🤦‍♂️

I'll get my coat! 🚪
